modules/runner/src/integrationTest/java/org/apache/ignite/internal/configuration/ItDistributedConfigurationPropertiesTest.java

```diff
@@ -228,7 +228,7 @@ void start() {
             Stream.of(clusterService, raftManager, cmgManager, metaStorageManager)
                     .forEach(IgniteComponent::start);
 
-            distributedCfgManager.start();
+            CompletableFuture.runAsync(distributedCfgManager::start);
```

Comment:
DistributedConfigurationManager's start is blocking, that's why I run it async

Reply:
But we start everything synchronously, what's the point? Or do you mean that DistributedConfigurationManager's start is dependent on some other actions and it blocks forever otherwise?

Reply:
Yes, exactly